website/source/docs/config/index.html.md

```diff
@@ -111,6 +111,22 @@ For Consul, the following options are supported:
 
   * `token` (optional) - An access token to use to write data to Consul.
 
+  * `tls_ca_file` (optional, mandatory when scheme is https) - The path to the CA certificate
```

Comment:
@armon I'm not sure I understand what you mean by mutual TLS is not done but we can verify the server authenticity.